pulp-glue/pulp_glue/common/context.py

```diff
@@ -742,7 +747,10 @@ def __init__(self, pulp_ctx: PulpContext, repository_ctx: "PulpRepositoryContext
 
     @property
     def scope(self) -> Dict[str, Any]:
-        return {self.repository_ctx.HREF: self.repository_ctx.pulp_href}
+        if self.repository_ctx.tangible:
+            return {self.repository_ctx.HREF: self.repository_ctx.pulp_href}
+        else:
+            return {}
```

Comment:
Now i need to find a new way to make the existence of lookup values in the repository context mandatory for version detail commands. XD

Reply:
When does `scope` get called? You could add more logic to the version generic command that checks `list_only` and sets a parameter on the context object which `scope` could check to make sure the lookup has been set.

Reply:
Yeah, something like that.

Reply:
It sounds easier to not use the baseclass RepositoryContext for the generic endpoint, but subclass one for it. I will keep the tangible however. I like it.